### ISSUE TITLE
show combined boxes Over under when direction Top/Bottom ist selected

### ISSUE THREAD

**Issue by michamarti** [Neriderc/GVExport#452]
for my yearly printout,
i find it convenient to have the family boxes in combined mode in Over under style for TopDown exports,
as width to height ratio improves a bit.

please discuss if that's worth incorporating, or should have a separate option, or should only live in my pull request
it's easy done with 
```
                          // Print wife
                          if (!empty($wife_id)) {

                             if ($this->settings["graph_dir"] != "LR" && !empty($husb_id)) {
                               $out .= "</TR>";
                               $out .= "<TR>";
                             }
                             $person = new Person([], $this);
                             $out = $person->addPersonLabel($wife_id, $out);
                          }
```
in Dot.php printFamily..

**Comment by Neriderc** [comment]
I think this should be an option not the only way of displaying it, but I think it's good to have this option. You're right, Top-to-bottom diagrams are very wide, and this helps improve that to some extent.

I've taken a screen shot of how it looks so others can comment on what they think:

![image](https://github.com/Neriderc/GVExport/assets/89974636/c4b593e9-6751-4864-8aff-822428182344)

I think it would be best to keep the default setting the current one that people are used to, but add this as an option they can change it to. That way we aren't changing what people are familiar with. I'm thinking when you choose the Combined mode, an option appears underneath to choose Side by side or Over under style.

Anyway, I'd be happy to include this as an option so long as no one has any strong views against it (which I can't imagine, since it would just be an option). Feel free to create a pull request to submit the code as you have it, and I'll handle adding the option to the form.

**Comment by Neriderc** [comment]
Talking to anyone who sees this: what are some opinions on how to put this in the UI? 

When you choose the "Side by side" option, it will show as it does now. Unselect, and it will show over/under.

![image](https://github.com/Neriderc/GVExport/assets/89974636/4f714aab-70f3-4bc3-8750-aceb9d94a867)


Or radio buttons:
![image](https://github.com/Neriderc/GVExport/assets/89974636/8bf9c424-b73f-4a70-8246-1dab06b8bb9f)

I did the radio buttons first, but felt it was a bit hard to follow having radio buttons so close to the ones above it.

Is the top one ok? I'm not particularly happy with either, so I'd be happy to hear some opinions on this.

**Comment by michamarti** [comment]
I'd prefer the Radio Button Version, cause you can see what you get (Over/Under),
whereas just switching Side by side checkbox off, you have to guess/know that you get Over/Under.

**Comment by Neriderc** [comment]
You're right, that's a good reason to use the radio buttons. 

I still don't like the position, and so I looked up some user experience information and it's recommended not to nest radio buttons.

I'm thinking now that the option should not be in this place, but rather down in the diagram option section here:
![image](https://github.com/Neriderc/GVExport/assets/89974636/fe7276c9-5285-4b35-9fc2-db130e19daf5)

I can add a new sub-section here and put in the option along with a note that it only applies to combined mode (we do this in other places too, having options that only apply under certain circumstances).

This setup will help keep the standard layout clean, but add this as an advanced option for people who want that control.

**Comment by michamarti** [comment]
very good, i fully agree 👍 
so you could also use Over/Under in combined L/R Layout, if needed for whatever reasons

**Comment by Neriderc** [comment]
Which looks better?

Option 1:
![image](https://github.com/Neriderc/GVExport/assets/89974636/d6c8374b-3ba9-4f1c-9948-5711ebf5505c)

Option 2:
![image](https://github.com/Neriderc/GVExport/assets/89974636/1cf6d0e5-e18d-495e-911c-81d3c18f90c7)

**Comment by Neriderc** [comment]
Third option, I think maybe this one is best?

![image](https://github.com/Neriderc/GVExport/assets/89974636/83b36f85-1b01-4598-8962-273968e2e86c)

**Comment by hartenthaler** [comment]
I prefer option 3.

**Comment by Neriderc** [comment]
I've made a few tweaks. We need to mention it only applies to the combined diagram type, and instead of Over / Under I've used Stacked as I think this conveys it a bit nicer.

I've also updated the help for this section. This involved splitting the help page into components, copying existing translations over, then adding new ones for the settings group name, settings name, settings values, descriptions, etc. These are updated in POEditor waiting to be translated, and I'll grab those translations at some point before the next release (once translators have had a chance to update).

Here's what I have. It's not too late, let me know if you think something needs changing.

![image](https://github.com/Neriderc/GVExport/assets/89974636/9febb112-1951-48f6-8000-be6adadf574e)
